### DISCUSSION TITLE
using SELECT with use-input clear appended used string in select box

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [quasarframework/quasar#12706] [General - Components / Directives / etc] [answered]
When I type something and pick the option from drop down, the thing I typed still remains in `SELECT` how can I clear it once the options is selected from dropdown?

```html
<q-select filled v-model="model" clearable use-input multiple counter use-chips label="Social Networks" :options="networks"></q-select>
```
<img width="587" alt="Screenshot 2022-03-08 at 10 48 24 PM" src="https://user-images.githubusercontent.com/8083613/157291306-45a4dca3-d468-4348-b5a7-4ff31fd7b8f4.png">

**Comment by MilosPaunovic** [comment, marked as answer]
You can achieve this using `refs` and `@update:model-value` event, like this:
[https://codepen.io/milospaunovic/pen/BameGba?editors=1011](https://codepen.io/milospaunovic/pen/BameGba?editors=1011)

**Reply by AbhimanyuAryan** [reply]
Thanks a lot @MilosPaunovic but sorry to bother you again. Is it possible for you to provide v1 syntax? We use Quasar Framework v1.15.10 UMMD and don't support {refs} yet

**Reply by metalsadman** [reply]
instead of `@update:model-value` you use `@input`. ie. `<q-select ref="mySelect" @input="$refs.mySelect.updateInputValue('')" ... />`

**Reply by AbhimanyuAryan** [reply]
I don't think there's `refs` in 1.15.10 I believe refs were introduced in vue3 @metalsadman

**Reply by metalsadman** [reply]
there's always been `$refs` in vue 2.

**Reply by AbhimanyuAryan** [reply]
@metalsadman ok let me check. Thanks

**Reply by metalsadman** [reply]
@AbhimanyuAryan https://v2.vuejs.org/v2/api/?redirect=true#vm-refs
https://v2.vuejs.org/v2/guide/components-edge-cases.html?redirect=true#Accessing-Child-Component-Instances-amp-Child-Elements

**Reply by AbhimanyuAryan** [reply]
thanks
